lib/manifestNoCatalogs.js

```diff
@@ -0,0 +1,21 @@
+const consts = require('./consts')
+
+const pkg = require('../package')
+
+module.exports = {
```

Comment:
```
const manifest = require('./manifest')

const manifestNoCatalogs = Object.assign({}, manifest)

manifestNoCatalogs.name += ' (without catalog support)'

manifestNoCatalogs.catalogs = []

manifestNoCatalogs.resources = manifest.resources.filter(resource => (resource.name != 'catalog'))

module.exports = manifestNoCatalogs
```

and in `./lib/manifest.js` we change `resources: ['catalog', ...]` to `resources: [{ name: 'catalog', types: ['other']}, ...]`

this will save us in the future... as we might edit `./lib/manifest.js` and forget to edit `./lib/manifestNoCatalogs.js` too